cmake/Modules/FindTPLLIBQUADMATH.cmake

```diff
@@ -1 +1,17 @@
-KOKKOS_FIND_IMPORTED(LIBQUADMATH HEADER quadmath.h LIBRARY quadmath)
+# gcc bundles libquadmath and doesn't need any extra libraries or include
```

Comment:
```suggestion
# gcc bundles libquadmath and doesn't need any extra link or include
```
to match better with the cmake vocabulary (`target_{link, include}_directories`).
I got confused with the initial wording and was wondering why you were explicitly linking against libquadmath below even though the comment said we don't need any extra librar[y]ies.

Reply:
[`f1eae63` (#5286)](https://github.com/kokkos/kokkos/pull/5286/commits/f1eae639c3efacf4b6837111fcd5eca38da18f97)